components/expenses/Expense.js

```diff
@@ -172,7 +175,7 @@ class Expense extends React.Component {
       await this.props.refetch();
     } catch (err) {
       console.error(err);
-      this.setState({ showDeleteExpenseModal: false, error: err.message });
+      this.setState({ showDeleteExpenseModal: false, error: err.message }, this.handleErrorMessage(err.message));
```

Comment:
This doesn't really makes sense to me:
- `state.error` is already set here, why do we need to call `this.handleErrorMessage(err.message)`?
- `this.handleErrorMessage(err.message)` returns undefined, so we're passing `undefined` as a callback to `setState`, why?

Reply:
Sorry, looks like I got ahead of myself here while I was trying to find a solution & ended up going in circles. This was unnecessary.